website/views/shared/tasks/index.jade

```diff
@@ -7,22 +7,22 @@ include ./task-list
 include ./task
 
 script(id='templates/habitrpg-tasks.html', type="text/ng-template")
-  .tasks-lists.container-fluid
+  .tasks-lists.container-fluid    
     .row
       .col-sm-6.col-md-3(
         ng-repeat='list in lists',
         ng-class='::{ "rewards-module": list.type==="reward", "new-row-sm": list.type==="todo" }')
         .task-column(class='{{::list.type}}s')
           include ./task_view/graph
-
           h2.task-column_title(class='{{::list.type}}-title') {{::list.header}}
+          h5.tasks-warning When you add a new task, it will be immediately saved to the challenge and given to all participants. Clicking the cancel button above will not revert your change.
```

Comment:
Oh yes, good point. @helenyau0 If you're not sure how to do localised text (i.e., text that can be translated into the user's own language), have a look at line 28 inn this file, where it says `=env.t('dailiesRestingInInn')`. That's an example of how to insert text in a way that can be translated. `dailiesRestingInInn` is a string in [website/common/locales/en/tasks.json](https://github.com/HabitRPG/habitica/blob/develop/website/common/locales/en/tasks.json). For this string that you'd need to add, the `challenges.json` file in the same directory would be appropriate.

Also, can you please remove the blank spaces or tabs at the end of lines 10 and 25? You'll see them highlighted in dark green in [GitHub's diff](https://github.com/HabitRPG/habitica/pull/8723/files) for your change. It's a minor point but it helps to keep the code clean.